CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- MySQL - Handle BIT(1) boolean columns and handle the buffer<01> value returned on join queries. 
```

Comment:
[Misc] Is this really a fix ? Looks like a change to me.

Reply:
Not really actually. We are supposed to handle booleans on mysql since ever. And for some reason it doesn't work today. 

Moreover, the mysql boolean support was pretty bad so far, especially on queries using joins. 

IMO it should stay on the fixed section. What would be the conclusion to you ?